docs/serving/autoscaling/autoscaling-metrics.md

```diff
@@ -15,52 +15,89 @@ The possible metric types that can be configured per revision depend on the type
 For more information about KPA and HPA, see the documentation on [Supported Autoscaler types](autoscaler-types.md).
 
 * **Per-revision annotation key:** `autoscaling.knative.dev/metric`
-* **Possible values:** `"concurrency"`, `"rps"` or `"cpu"`, depending on your Autoscaler type. The `cpu` metric is only supported on revisions with the HPA class.
+* **Possible values:** `"concurrency"`, `"rps"`, `"cpu"`, `"memory"` or any custom metric name, depending on your Autoscaler type. The `cpu`/`memory`/`custom` metrics are only supported on revisions with the HPA class.
 * **Default:** `"concurrency"`
 
 
-=== "Per-revision concurrency configuration"
-    ```yaml
-    apiVersion: serving.knative.dev/v1
-    kind: Service
+=== Per-revision concurrency configuration
```

Comment:
@snneji can you confirm whether it was formatted like this before for tabs or something? I can't find it documented in the style guide.

Reply:
Yes this used to be formatted as tabs. The 4 spaces of indentation these code snippets used to have needs to added back for the formatting to render correctly.